fastembed/image/transform/operators.py

```diff
@@ -59,6 +59,9 @@ def __init__(self, scale: float = 1 / 255):
     def __call__(self, images: List[np.ndarray]) -> List[np.ndarray]:
         return [rescale(image, scale=self.scale) for image in images]
 
+class PILtoNDarray:
+    def __call__(self, images: List[Union[Image.Image, np.ndarray]]) -> List[np.ndarray]:
+        return [np.asarray(image).swapaxes(2, 0) if isinstance(image, Image.Image) else image for image in images]
```

Comment:
it seems to be 

(H, W, C) -> (C, W, H)

but should be (H, W, C) -> (C, H, W)

so we need to use transpose((2, 0, 1)) instead of swapaxes, should not we?

Reply:
But it's literally the same thing?

Reply:
```python
a = np.random.random((3,4,5))
b = a.swapaxes(2, 0)
c = a.transpose((2, 0, 1))
print(a.shape, b.shape, c.shape)
>>> ((3, 4, 5), (5, 4, 3), (5, 3, 4))
```